### DISCUSSION TITLE
How can I minimize trilium to tray on Windows？

### DISCUSSION THREAD

**Discussion by 1ridic** [TriliumNext/Trilium#2627] [Q&A] [answered]
I tried to minimize but the icon does not appear on tray bar. Is this the feature that needs to be turned on to use it?

**Comment by zadam** [comment, marked as answer]
Hi, it should work out of the box. In Windows there's few tray icons directly visible and then the rest is hidden by default, this should be configurable in the windows taskbar settings.

**Reply by 1ridic** [reply]
Found it, many thanks.
